event_server.cpp

```diff
@@ -1861,6 +1861,36 @@ static void get_settling(JObj& response, const json_value *params)
     response << jrpc_result(settling);
 }
 
+static void get_variable_delay_settings(JObj& response, const json_value* params)
+{
+    JObj rslt;
+
+    VarDelayCfg delayParams = pFrame->GetVariableDelayConfig();
+    rslt << NV("Enabled", delayParams.enabled)
+        << NV("ShortDelay", delayParams.shortDelay / 1000)
+        << NV("LongDelay", delayParams.longDelay / 1000);

```

Comment:
I'm guessing the intent is to output fractional seconds (floating point), not just integer seconds, right?
if so, we should make these into floating point calculations:
```suggestion
        << NV("ShortDelay", (double) delayParams.shortDelay / 1000., 3)
        << NV("LongDelay", (double) delayParams.longDelay / 1000., 3);
```